src/js/components/Loader.js

```diff
@@ -0,0 +1,86 @@
+import classNames from 'classnames/dedupe';
+import React from 'react';
+
+/*
+ * Example usage:
+ * <Loader className="inverse" type="ballSpinFadeLoader" />
+ */
+
+// Available types:
+let typeMap = {
+  ballBeat: {className: 'ball-beat', divCount: 3},
+  ballClipRotate: {className: 'ball-clip-rotate', divCount: 1},
+  ballClipRotateMultiple:
+    {className: 'ball-clip-rotate-multiple', divCount: 2},
+  ballClipRotatePulse: {className: 'ball-clip-rotate-pulse', divCount: 2},
+  ballGridBeat: {className: 'ball-grid-beat', divCount: 9},
+  ballGridPulse: {className: 'ball-grid-pulse', divCount: 9},
+  ballPulseRise: {className: 'ball-pulse-rise', divCount: 5},
+  ballPulseSync: {className: 'ball-pulse-sync', divCount: 3},
+  ballRotate: {className: 'ball-rotate', divCount: 1},
+  ballScale: {className: 'ball-scale', divCount: 1},
+  ballScaleMultiple: {className: 'ball-scale-multiple', divCount: 3},
+  ballScaleRandom: {className: 'ball-scale-random', divCount: 3},
+  ballScaleRipple: {className: 'ball-scale-ripple', divCount: 1},
+  ballScaleRippleMultiple:
+    {className: 'ball-scale-ripple-multiple', divCount: 3},
+  ballSpinFadeLoader: {className: 'ball-spin-fade-loader', divCount: 8},
+  ballTrianglePath: {className: 'ball-triangle-path', divCount: 3},
+  ballZigZag: {className: 'ball-zig-zag', divCount: 2},
+  ballZigZagDeflect: {className: 'ball-zig-zag-deflect', divCount: 2},
+  cubeTransition: {className: 'cube-transition', divCount: 2},
+  lineScale: {className: 'line-scale', divCount: 5},
+  lineScaleParty: {className: 'line-scale-party', divCount: 4},
+  lineScalePulseOut: {className: 'line-scale-pulse-out', divCount: 5},
+  lineScalePulseOutRapid:
+    {className: 'line-scale-pulse-out-rapid', divCount: 5},
+  lineSpinFadeLoader: {className: 'line-spin-fade-loader', divCount: 8},
+  pacman: {className: 'pacman', divCount: 5},
+  semiCircleSpin: {className: 'semi-circle-spin', divCount: 1},
+  triangleSkewSpin: {className: 'triangle-skew-spin', divCount: 1}
```

Comment:
That's understandable. I guess I'd prefer to remove them and link to the repository of available options, but I don't care that much — if you'd prefer to comment them out, let's do that!